module/VuFind/sql/mysql.sql

```diff
@@ -346,3 +346,29 @@ CREATE TABLE `auth_hash` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE utf8mb4_unicode_ci;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
+--
+-- Table structure for table `feedback`
+--
+
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8mb4 */;
+CREATE TABLE `feedback` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `user_id` int(11) DEFAULT NULL,
+  `message` longtext,
+  `form_data` json DEFAULT NULL,
+  `form_name` varchar(255) NOT NULL,
+  `created` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `updated` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `updated_by` int(11) DEFAULT NULL,
+  `status` enum('open','in progress','pending','answered','closed') NOT NULL DEFAULT 'open',
```

Comment:
Have you tested whether VuFind's database upgrade tool can handle enums correctly? I'm not sure if we've encountered this case before. (I haven't taken the time to try it yet -- my guess is that it will probably work when the table is added for the first time, but it might be problematic if we change the table in the future; especially if we change the values in the enum).

Reply:
I tested it now and it looks like it is correctly handled. Adding new table was no problem, then I modified the enum values - add some and remove some and it worked well.

Reply:
Thanks for taking the time to confirm!